src/server.js

```diff
@@ -0,0 +1,40 @@
+const FeedParser = require('feedparser')
+const request = require('request')
+const Rx = require('rx') 
+
+// Event loop will run 3 times for testing
+const interval = Rx.Observable.interval(5000).timeInterval().take(3)
```

Comment:
Additionally, it would be better if this was configurable. Eventually, you should add a config that extends the config in `windbreaker-service-util`.